services/management/dbaas/components_service.go

```diff
@@ -105,6 +176,10 @@ func (c componentsService) versions(ctx context.Context, params componentsParams
 
 	versions := make([]*dbaasv1beta1.Version, 0, len(components.Versions))
 	for _, v := range components.Versions {
+		err = c.removeDisabledVersions(params.operator, v.Matrix)
```

Comment:
My suggestion would be to just pass minimum version into `c.matrix` and take care it there.
For example: `c.matrix(v.Matrix.Mongod, "4.2.0")`

Reply:
We should be able to filter out single version.

Reply:
Not in the scope of this ticket

Reply:
Cool, at least It will be simple.